eng/CodeChecks.ps1

```diff
@@ -72,18 +72,19 @@ try {
         & $PSScriptRoot\Update-Snippets.ps1 @script:PSBoundParameters
     }
 
-    Write-Host "Re-generating lisgings"
+    Write-Host "Re-generating listings"
     Invoke-Block {
         & $PSScriptRoot\Export-API.ps1 @script:PSBoundParameters
     }
 
     Write-Host "Re-generating clients"
     Invoke-Block {
-        # https://github.com/Azure/azure-sdk-for-net/issues/8584
-        # & $repoRoot\storage\generate.ps1
+        & $repoRoot\storage\generate.ps1
     }
 
     Write-Host "git diff"
+    # prevent warning related to EOL differences which triggers an exception for some reason
+    & git config core.safecrlf false
```

Comment:
I'm not sure I like this script automatically changing someone's local git config. If we want this option just for the git diff call pass the config directly with that command instead.

Reply:
I don't think there is a way to pass this as an argument to diff call. I can at least just lookup and then restore the value of core.safecrlf.